include/api/debug.h

```diff
@@ -123,14 +123,18 @@ static inline void debug_printTCB(tcb_t *tcb)
 
 static inline void debug_dumpScheduler(void)
 {
-    printf("Dumping all tcbs!\n");
-    printf("Name                                    \tState          \tIP                  \t Prio \t Core%s\n",
-           config_set(CONFIG_KERNEL_MCS) ?  "\t InReleaseQueue" : "");
```

Comment:
Are you able to move this commit to a separate PR. It's more of a subjective discussion whether reformatting this debug syscall is wanted.  In particular removing the `InReleaseQueue` state and adding ANSI color codes is not an unambiguous improvement and discussing that is a different scope than the domain scheduler changes..

Reply:
Also, we could merge this change then with what I have pending in https://github.com/seL4/seL4/pull/665/commits/1f3c02f540293fad9e0dbed37a32df8dcb6e499f that dynamically adapts the numer of columns to the the configuration (MCS, SMP)

Reply:
Yes, we could probably split this up into two PRs. I didn't think a debug print would have that much contention, but since it seems to, yes, we can move it.